graphbolt/src/cnumpy.cc

```diff
@@ -0,0 +1,226 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file cnumpy.cc
+ * @brief Numpy File Fetecher class.
+ */
+
+#include "cnumpy.h"
+
+#include <torch/torch.h>
+
+#include <cstring>
+#include <regex>
+#include <stdexcept>
+
+namespace graphbolt {
+namespace storage {
+
+constexpr int disk_alignment_size = 4096;
+
+c10::intrusive_ptr<OnDiskNpyArray> CreateDiskFetcher(
+    std::string path, torch::ScalarType dtype) {
+  return c10::make_intrusive<OnDiskNpyArray>(path, dtype);
+}
```

Comment:
Make it a static class function named `Create` inside the OnDiskNpyArray?
Similar to https://github.com/dmlc/dgl/blob/2bda158238371231452c8449a4b144f1c736b786/graphbolt/src/python_binding.cc#L84C10-L84C34
and
https://github.com/dmlc/dgl/blob/2bda158238371231452c8449a4b144f1c736b786/graphbolt/include/graphbolt/fused_csc_sampling_graph.h#L100